third_party/mbedtls/mbedtls-config.h

```diff
@@ -28,11 +28,18 @@
 #ifndef MBEDTLS_CONFIG_H
 #define MBEDTLS_CONFIG_H
 
-#include <inttypes.h>
 #include <stdlib.h>
 
 #include <platform/logging.h>
 
+#if defined(_WIN32)
```

Comment:
Not without editing the mbedtls code directly. I don't have permission to edit that project, just OpenThread. So until I get permission, I can't fix these the correct way.